up42/auth.py

```diff
@@ -153,6 +154,36 @@ def _endpoint(self) -> str:
         """Gets the endpoint."""
         return f"https://api.up42.{self.env}"
 
+    def _get_auth_token(self):
```

Comment:
Do we need this? I think we can obtain this from the auth class.

With these two lines, you make sure the auth is authenticated and get the token. That makes everything you do in this function. However, by using the auth._request you would avoid this.

:) 
 
auth_class._get_token()
return auth_class.token

Reply:
Indeed!
Removing this method altogether. Thanks :)